.claude/skills/allocation-diff/scripts/compute_diff.py

```
@@ -0,0 +1,100 @@
+# ---------------------------------------------------------------------------
+
+def extract(data):
+    return {k: v["locationSize"] for k, v in data["data"].items()}
+
+
+def try_load(source, subdir, fname):
+    try:
+        return extract(source.load(subdir, fname))
+    except (FileNotFoundError, subprocess.CalledProcessError):
+        return None
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)